extension/settingsproviderextension/testdata/settings.yaml

```diff
@@ -0,0 +1,20 @@
+---
+- createdTimeStamp: 1723485600
+  id: "1122334455"
```

Comment:
We didn't think about the config file based setup when creating the data types, but having a `createdTimestamp`, `shard`, and `id`  is pretty awkward :grimacing: . I guess not a big problem since we'll likely only use it for tests.

Even the `type` is a bit funny. In a normal configuration file we'd probably split it over 2 keys:
```
otelComponentMappings:
  - ...
  - ...
otelRelationMappings:
 - ...
 - ...
 ```